src/main/java/jeryl/fyp/model/student/exceptions/DuplicateStudentException.java

```diff
@@ -0,0 +1,11 @@
+package jeryl.fyp.model.student.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Students (Students are considered duplicates if they have the
+ * same identity).
+ */
+public class DuplicateStudentException extends RuntimeException {
+    public DuplicateStudentException() {
+        super("Operation would result in duplicate persons");
```

Comment:
'duplicate students' to sync up the rest

Reply:
Good catch! Fixed.